internal/smbsafe/smbsafe_test.go

```diff
@@ -54,7 +54,7 @@ func TestExclusiveLockSmb(t *testing.T) {
 	}()
 
 	// Wait more than minimum duration time to release the lock
-	time.Sleep(waitTime)
+	time.Sleep(waitTime + time.Millisecond)
```

Comment:
Instead of increasing the `time.Sleep` here, I would rather modify shouldHaveWaited and shouldNotHaveWaited to remove this millisecond. That way, all tests depending on it are impacted

Reply:
I agree. It's better to change directly in the helpers. It's updated now! Increased the thresholds by `+/- time.Millisecond` (`+` for comparisons where it should be higher and `-` for the ones where it should be lower)